packages/react-dom/src/client/ToStringValue.js

```diff
@@ -28,6 +29,7 @@ export function getToStringValue(value: mixed): ToStringValue {
     case 'number':
     case 'object':
     case 'string':
+    case 'bigint':
```

Comment:
nit-pick: Both the opaque type and case statement are ordered A-Z.  Thus, should be move bigint into its natural place? e.g.

```
    case 'bigint':
    case 'boolean':
    case 'number':
    case 'object':
    case 'string':
```

Reply:
Alright. Will do.